Library/Homebrew/extend/ARGV.rb

```diff
@@ -268,6 +268,10 @@ def fetch_head?
     include? "--fetch-HEAD"
   end
 
+  def ignore_installed_specs?
+    include? "--ignore-installed-specs"
```

Comment:
@ilovezfs so, both with supporting the specs we need to think of the way we trigger builds when dependency is satisfied, but not stable, right?

E.g.

`go-1.7rc5` installed and upgraded to `go-1.7rc6` while installing `godep`, but `godep` need to be built from source here, right?